src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/model/Innsatsgruppe.kt

```diff
@@ -5,4 +5,5 @@ enum class Innsatsgruppe {
     SITUASJONSBESTEMT_INNSATS,
     SPESIELT_TILPASSET_INNSATS,
     VARIG_TILPASSET_INNSATS,
+    GRADERT_VARIG_TILPASSET_INNSATS,
```

Comment:
Hvis du legger til denne må vi kanskje vurdere å endre i amt-person også, for der mapper vi denne til VARIG_TILPASSET_INNSATS fordi de skal behandles likt hos oss.